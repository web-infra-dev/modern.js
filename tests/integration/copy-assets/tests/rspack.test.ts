import { testPublicHtml } from './testPublicHtml';

describe('copy assets', () => {
  test(`should copy public html and replace the assetPrefix variable in rspack`, async () => {
    await testPublicHtml('rspack');
  });
});
