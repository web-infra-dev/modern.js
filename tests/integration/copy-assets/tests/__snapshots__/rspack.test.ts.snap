// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`copy assets should copy public html and replace the assetPrefix variable in rspack 1`] = `
"<!DOCTYPE html>
<html>
  <head>
    <script>
      window.__assetPrefix__ = 'https://demo.com';
      window.__assetPrefix2__ = 'https://demo.com';
      window.__assetPrefix3__ = 'https://demo.com';
    </script>
  </head>
</html>
"
`;
